components/suggestion/switch_channel_provider.jsx

```diff
@@ -151,7 +133,7 @@ class SwitchChannelSuggestion extends Suggestion {
                 {...Suggestion.baseProps}
             >
                 {icon}
-                {displayName}
+                <span style={this.displayNameStyle}>{displayName}</span>
```

Comment:
This actually fixes a second bug where long display names would cause the GM icon to get squished